website/docs/r/nsxt_edgegateway.html.markdown

```diff
@@ -0,0 +1,177 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_nsxt_edgegateway"
+sidebar_current: "docs-vcd-resource-nsxt-edge-gateway"
+description: |-
+  Provides a VMware Cloud Director NSX-T edge gateway. This can be used to create and delete NSX-T edge gateways connected to external networks.
+---
+
+# vcd\_nsxt\_edgegateway
+
+Provides a VMware Cloud Director NSX-T edge gateway. This can be used to create and delete NSX-T edge gateways connected
```

Comment:
What about update? Isn't it supported?

Reply:
It does. Update is here. This is the problem of cloning old edge gateway docs. Updated docs.